dapp/src/utils/web3.js

```diff
@@ -1,7 +1,6 @@
 const networkInfo = {
   1: 'Mainnet',
   3: 'Ropsten',
```

Comment:
Let's also cleanup those other testnet and only keep Mainnet and Localhost

Reply:
Removed in the latest commit